src/codegen/utilities/attribute_helpers.py

```diff
@@ -94,12 +94,27 @@
         "deprecated_in": "0.6.6",
     },
     "over_write": {"new_name": "overwrite", "deprecated_in": "0.6.6"},
+    "expir_states_ao_type": {"new_name": "ao_output_type", "deprecated_in": "0.6.6"},
+    "expir_states_co_state": {"new_name": "co_state", "deprecated_in": "0.6.6"},
+    "expir_states_do_state": {"new_name": "do_state", "deprecated_in": "0.6.6"},
+    "expir_states_ao_state": {"new_name": "ao_state", "deprecated_in": "0.6.6"},
+    "expir_trig_trig_type": {"new_name": "expiration_trigger_type", "deprecated_in": "0.6.6"},
+    "expir_trig_dig_edge_edge": {
+        "new_name": "expiration_trigger_dig_edge_edge",
+        "deprecated_in": "0.6.6",
+    },
+    "expir_trig_dig_edge_src": {"new_name": "expiration_trigger_type", "deprecated_in": "0.6.6"},
```

Comment:
The new name is incorrect.
```suggestion
    "expir_trig_dig_edge_src": {"new_name": "expiration_trigger_dig_edge_src", "deprecated_in": "0.6.6"},
```